packages/components/src/components/contract-card/contract-card-items/__tests__/multiplier-card-body.spec.tsx

```diff
@@ -91,19 +93,22 @@ describe('MultiplierCardBody', () => {
         delete mock_props.contract_info.cancellation;
         delete mock_props.contract_info.sell_price;
 
+        // @ts-expect-error Check if error is gone after migrating MultiplierCardBody to TS
```

Comment:
Type error about MultiplierCardBody props type appeared here after I spread the props inside it. Needs to be checked after its TS migration.